sway-ir/src/optimize/simplify_cfg.rs

```diff
@@ -33,43 +31,62 @@ pub fn simplify_cfg(context: &mut Context, function: &Function) -> Result<bool,
 }
 
 fn remove_dead_blocks(context: &mut Context, function: &Function) -> Result<bool, IrError> {
-    // Find a block other than 'entry' which has no predecessors, remove it if found.
-    function
-        .block_iter(context)
-        .skip(1)
-        .find(|block| block.num_predecessors(context) == 0)
-        .map(|dead_block| function.remove_block(context, &dead_block))
-        .transpose()
-        .map(|result| result.is_some())
-}
-
-fn merge_blocks(context: &mut Context, function: &Function) -> Result<bool, IrError> {
-    // Get the block a block branches to iff its terminator is `br`.
-    let block_terms_with_br = |from_block: Block| -> Option<(Block, Block)> {
-        from_block.get_terminator(context).and_then(|term| {
-            if let Instruction::Branch(to_block) = term {
-                Some((from_block, *to_block))
-            } else {
-                None
+    let mut worklist = std::vec::Vec::<Block>::new();
```

Comment:
`Vec` is in the prelude, you shouldn't need to fully qualify it here.  I'm surprised `clippy` doesn't mention it.

Reply:
> `Vec` is in the prelude, you shouldn't need to fully qualify it here. I'm surprised `clippy` doesn't mention it.

I just fixed this. Please re-approve.